tests/unit/amo/pages/TestCollectionList.js

```diff
@@ -101,10 +109,12 @@ describe(__filename, () => {
     );
 
     fakeDispatch.resetHistory();
+    const errorHandler = createStubErrorHandler();
```

Comment:
Now that you are using `neverCalledWithMatch`, you shouldn't need to create and pass in an `errorHandler`.